src/client/containers/CreateTeamPage/CreateTeamPage.component.js

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import HeaderTitle from '../../components/Title/Title';
+import Footer from '../../components/FooterComponent/FooterComponent';
+import AddAvatarComponent from '../../components/AddAvatarComponent/AddAvatarComponent';
+import { FormContainer } from '../FormContainer/FormContainer.component';
 
 export const CreateTeamPage = () => (
-  <HeaderTitle title="This is the Create Team Page" />
+  <div id="root">
```

Comment:
This is the div that react uses to mount itself. You don't want to replicate that inside of React. I don't think it does any harm but also does not serve a purpose.